src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/CassandraEventsSink.scala

```diff
@@ -65,10 +68,26 @@ object CassandraEventsSink{
           aggregators.foreach(aggregator => {
             val eventName = aggregator.FortisTargetTablename
 
-            Timer.time(Telemetry.logSinkPhase(s"aggregate_$eventName", _, _, batchSize)) {
+            Timer.time((completed, time)=> println(s"Ran aggregator ${aggregator.getClass.getSimpleName} in ${time} nanoseconds.")) {
```

Comment:
Nit: spacing ` => `

Reply:
Oops, good catch - I actually meant leave the original Telemetry call!